x/wasm/keeper/querier_test.go

```diff
@@ -503,6 +528,7 @@ func TestQueryContractHistory(t *testing.T) {
 			}
 			require.NoError(t, err)
 			assert.Equal(t, spec.expContent, got.Entries)
+			fmt.Println(base64.StdEncoding.EncodeToString(got.Pagination.NextKey))
```

Comment:
Shouldn't this be tested with assert ~?

Reply:
Thanks, I forgot to delete it. But I don't think it is necessary to perform an assertion. If the nextKey is incorrect, the pagination result will also be inaccurate.